.isort.cfg

```diff
@@ -0,0 +1,10 @@
+[settings]
+profile=black
+known_first_party=spack
+known_archspec=archspec
+known_ruamel=ruamel
+known_compat=compat
+known_llnl=llnl
+known_third_party=six
+known_typing=typing,typing_extensions
+sections=FUTURE,STDLIB,TYPING,THIRDPARTY,COMPAT,ARCHSPEC,RUAMEL,LLNL,FIRSTPARTY,LOCALFOLDER
```

Comment:
Our isort configuration is now stored in `pyproject.toml`, so this can be removed.

Reply:
Thanks! Done.